rest/data/commit_queue.go

```diff
@@ -318,6 +318,28 @@ func EnqueuePRToCommitQueue(ctx context.Context, env evergreen.Environment, sc C
 		return nil, errors.New("PR contains no base branch label")
 	}
 
+	if !thirdparty.IsUnblockedGithubStatus(*pr.MergeableState) {
```

Comment:
can we use GetMergeableState() to avoid nil pointer issues

Reply:
done